org.kde.isoimagewriter.json

```
@@ -0,0 +1,34 @@
+{
+   "id": "org.kde.isoimagewriter",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "5.15-22.08",
+   "sdk": "org.kde.Sdk",
+   "command": "isoimagewriter",
+   "finish-args": [
+      "--device=dri",
+      "--filesystem=host",
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--system-talk-name=org.freedesktop.UDisks2"
+   ],
+   "modules": [
+      {
+         "name": "isoimagewriter",
+         "buildsystem": "cmake-ninja",
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://download.kde.org/stable/isoimagewriter/1.0.0/isoimagewriter-1.0.0.tar.xz",
+               "sha256": "a69022303ec1beb6b7b4f0d68e59e4199d3c9868767cb9eb23c6dd1999e06a5d",
+               "x-checker-data": {
+                     "type": "anitya",
+                     "project-id": 341536,
+                     "stable-only": true,
+                     "url-template": "https://download.kde.org/stable/isoimagewriter/$version/isoimagewriter-$version.tar.xz"
+                  }
+            }
```

```suggestion
            },
            {
                    "type": "patch",
                    "path": "logo.patch"
            }
```

with logo.patch being generated with `git show 62daf9a > logo.patch`